packages/dx-react-scheduler-material-ui/src/templates/appointment-tooltip/layout.test.jsx

```diff
@@ -121,14 +122,18 @@ describe('Appointment Tooltip', () => {
         .toBeTruthy();
     });
 
-    it('should render appointment dates', () => {
-      const tree = shallow((
-        <Layout {...defaultProps} />
+    it('should call dates format function', () => {
```

Comment:
We also need a test that ensures that this formatted data is actually rendered in text container

Reply:
We test it in `scheduler-core`. Here we tested only that format function is called.

Reply:
Calling a formatDate function doesn't automatically mean that formatted date will be rendered in the right place

Reply:
Yes, you are right!